go.mod

```diff
@@ -236,3 +236,5 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/onflow/flow-go v0.35.14-crescendo-preview.27.0.20240626210601-604590f19db9 => github.com/AndriiDiachuk/flow-go v0.0.0-20240711140457-69e2f4007091
```

Comment:
this will be replaced with an `onflow/flow-go` release in a follow up PR after merging the flow-go changes

Reply:
It would be great if we could avoid `replace` statements on `master`. It's too easy to forget to remove them again, and that happened a few times before.

Usually we do this by:
1. Opening a PR in flow-go
2. Opening a PR here in flow-emulator
3. Use the flow-emulator PR in the flow-go PR and merge the flow-go PR.
4. Update flow-emulator to the latest flow-go

Reply:
No, there is not replace statement added anywhere in the process above.

Reply:
@turbolent we typically do that for PRs that are created directly from the `onflow/flow-emulator` repo. In this case, the PR is from a fork.

We decided at one point to optimize for having only `onflow` commits in `flow-go`, and temporarily have a replace statement in the emulator repo while we merge the changes.